css/styles.css

```diff
@@ -0,0 +1,87 @@
+body {
+  background-color: #f0f0f0;
+}
+
+label,
+input,
+textarea {
+  display: block;
+  width: 100%;
+}
+
+input,
+textarea {
+  font-family: 'Open Sans', sans-serif;
+}
+
+textarea {
+  margin-bottom: 1rem;
+  border: 1.3px gray solid;
+  border-radius: 3px;
+  padding: 8px;
+}
+
+input,
+textarea:focus {
+  outline-color: #562b81;
+}
+
+img {
+  width: 100%;
+  border-radius: 7px;
+  margin-bottom: 10px;
+}
+
+/* Header */
+
+.header-nav {
+  background-color: #562b81;
+  height: 53px;
```

Comment:
This shouldn't be necessary. In general, you should avoid setting `height` on elements unless it is absolutely necessary. You want the elements on the page to be able to take up as much space as is needed. If your nav is too large without setting a height, you may just need to reduce the margin on the `h4` within it.